neat-form-core/src/test/java/com/nerdstone/neatformcore/robolectric/builders/MultiChoiceCheckBoxViewBuilderTest.kt

```diff
@@ -85,7 +89,8 @@ class `Test building MultiChoiceCheckBox view` : BaseJsonViewBuilderTest() {
     @Test
     fun `Should create multiple checkboxes when options are defined`() {
         val text = "Pick your programming languages"
-        viewProperty.viewAttributes = hashMapOf("text" to text)
+        val checkBoxOptionsTextSize = 22
+        viewProperty.viewAttributes = mutableMapOf("text" to text,"check_box_options_text_size" to checkBoxOptionsTextSize)
```

Comment:
Remember to change *check_box_options_text_size* to **options_text_size** if you do change the attribute name